lib/src/__tests__/DatePicker/Calendar.test.tsx

```diff
@@ -50,54 +50,3 @@ describe('Calendar - disabled selected date on mount', () => {
     expect(onChange).toHaveBeenCalledWith(utilsToUse.date('01-01-2018'), false);
   });
 });
-
-describe('Calendar - keyboard control', () => {
-  let component: ShallowWrapper;
-  const onChangeMock = jest.fn();
-
-  beforeEach(() => {
-    jest.resetAllMocks();
-    component = shallowRender(props => (
-      <Calendar
-        date={utilsToUse.date('01-01-2017')}
-        minDate={new Date('01-01-2018')}
-        onChange={onChangeMock}
-        theme={{ direction: 'ltr' } as any}
-        allowKeyboardControl
-        {...props}
-      />
-    ));
-  });
-
-  it('Should render go to prev week on up', () => {
```

Comment:
I am not really sure, do we have this tests in `e2e`. Anyway I planned to move them to cypress